server/chat-plugins/randombattles/index.ts

```diff
@@ -776,13 +779,19 @@ export const commands: Chat.ChatCommands = {
 		if (dex.gen === 1) {
 			const rbyMoves = getRBYMoves(species);
 			if (!rbyMoves) {
+				if (isInexact) {
+					return this.errorReply(`No Pok\u00e9mon named '${args[0].trim()}' was found. (Check your spelling?)`);
+				}
```

Comment:
I would probably shift this check up to right after the dataSearch call so that you only use it once. Right now, it's a bit confusing as to why it's in two places.

Reply:
I tried to best follow the logic from the original code.  It seemed as though there could be a case where the species exists, but doesn't have any movesets for gen1/lgpe.  I don't know enough about those formats to know the conditions for which that path gets triggered, though.  To keep it simple, I just removed the excess checks.

I've removed the extra check and left it as is in the latest commit.  Should be good to merge now.